.github/workflows/rust-crate-release.yml

```diff
@@ -17,27 +17,27 @@ jobs:
       image: wasmedge/wasmedge:ubuntu-build-clang
 
     steps:
-    - uses: actions/checkout@v2
-      with:
-        fetch-depth: 0
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
 
-    - name: Install Stable Rust
-      uses: actions-rs/toolchain@v1
-      with:
-        toolchain: stable
+      - name: Install Stable Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
 
-    - name: Build WasmEdge using clang
-      run: |
-        cmake -Bbuild -GNinja -DWASMEDGE_BUILD_SHARED_LIB=ON  -DCMAKE_INSTALL_PREFIX:PATH=/usr .
-        cmake --build build --target install
+      - name: Build WasmEdge using clang
+        run: |
+          cmake -Bbuild -GNinja -DWASMEDGE_BUILD_SHARED_LIB=ON  -DCMAKE_INSTALL_PREFIX:PATH=/usr .
+          cmake --build build --target install
 
-    - name: Publish wasmedge-sys crate
-      env:
-        CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
-      shell: bash
-      run: |
-        cd ./bindings/rust/wasmedge-sys
-        git config --global user.email "runner@gha.local"
-        git config --global user.name "Github Action"
-        cargo publish --dry-run
-        cargo publish
+      - name: Publish wasmedge-sys crate
+        env:
+          CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
+        shell: bash
+        run: |
+          cd ./bindings/rust/wasmedge-sys
+          git config --global user.email "runner@gha.local"
+          git config --global user.name "Github Action"
+          cargo publish --dry-run
+          cargo publish --token ${CARGO_REGISTRY_TOKEN}
```

Comment:
You don't need to specify the `CARGO_REGISTRY_TOKEN` when publishing to crates.io. The cargo tool will take the environment variable `CARGO_REGISTRY_TOKEN` directly.

Reply:
Accepted, I'll remove the option. Thanks!